setup-buildengine.sh

```diff
@@ -45,7 +45,7 @@ OPT_UPLOAD_PATH=$DEF_UPLOAD_PATH
 OPT_COMPRESSION=9
 OPT_RELEASE="latest"
 OPT_TOOLING="Jolla-latest-Sailfish_SDK_Tooling-i486.tar.bz2"
-OPT_TARGET_ARM="Jolla-latest-Sailfish_SDK_Target-armv7hl.tar.bz2"
+OPT_TARGET_ARM_ARMV7HL="Jolla-latest-Sailfish_SDK_Target-armv7hl.tar.bz2"
```

Comment:
The extra ARM there is just redundant. ARMv7hl already tells us that it's ARM.

Reply:
I wanted to group in $ARCH_$SUBARCH not so good idea?

Reply:
To me it feels unnecessary. Everyone knows Aarch64 is ARM and i486 is x86. And from the SDK point of view they are just different targets - it doesn't make any difference whether they are ARM or x86.